### ISSUE TITLE
Searing Gorge .gm off invisible npc

### ISSUE THREAD

**Issue by LeRutY** [ottconnect/Legion#101]
NPC: Overseer Oilfist 
NpcID: 14625

NPC is visible when .gm on, but not if .gm off. I know I could fix it myself, but when I type ingame: ".npc set phase 1" it says: "Uses invalid phaseID". In the database there is no "phase mask" but "phaseID", it´s set at 1.

Please help.

**Comment by ottconnect** [comment]
did you try setting phaseid 0, because the default phaseid is 0

**Comment by LeRutY** [comment]
Yes I tried phaseID "0" on all three entries (2 with 11895 HP and 1 with 1 HP) but none of them showed up. Still only if .gm on. I want to note: If .gm on there are 3 "Overseer Oilfist" but everyone has normal HP, I wonder where the one with 1 HP is.

"Hansel Heavyhands" (NpcID: 14627) is also affected, not visible if .gm off.
